CMakeLists.txt

```diff
@@ -1230,6 +1230,12 @@ if(ENABLE_HOST_CPU_DEVICES AND NOT ENABLE_CONFORMANCE)
     message(STATUS "libjpeg-turbo v3.0.0+ not found, disabled")
   endif()
 
+  find_package(OpenCV COMPONENTS dnn)
+  set(HAVE_OPENCV ${OpenCV_FOUND})
+  if(NOT ${HAVE_OPENCV})
+    message(STATUS "OpenCV not found, disabled")
+  endif()
+
```

Comment:
I wonder if we are starting to rack up too many dependencies. We could onnxruntime for performing the [NMS](https://onnx.ai/onnx/operators/onnx__NonMaxSuppression.html).